vllm/v1/structured_output/__init__.py

```diff
@@ -41,6 +40,7 @@ def _delayed_init(self):
         tokenizer_group.ping()
 
         tokenizer = tokenizer_group.get_lora_tokenizer(None)
```

Comment:
Not directly related to this PR but it looks like structured output currently isn't compatible with custom vocab LoRAs.